apps/typegpu-docs/src/components/design/Button.tsx

```diff
@@ -2,19 +2,18 @@ import cs from 'classnames';
 import { type ReactNode, forwardRef } from 'react';
 
 type Props = {
-  label?: string;
   accent?: boolean;
   onClick?: () => void;
   children?: ReactNode;
 };
 
 export const Button = forwardRef<HTMLButtonElement, Props>((props, ref) => {
-  const { label, onClick, accent, children } = props;
+  const { onClick, accent, children } = props;
 
   return (
     <button
       className={cs(
-        'focus:ring-2 focus:ring-gradient-blue rounded-[6.25rem] text-sm px-5 py-2.5',
+        'inline-flex justify-center items-center font-sans box-border focus:ring-2 focus:ring-gradient-blue rounded-[6.25rem] text-sm px-5 py-2.5',
```

Comment:
instead of using `font-sans` here I would add a global css reset to make buttons and inputs inherit font properties. now the `Select` component has a wrong font family